site/content/en/docs/administration/advanced/analytics.md

```diff
@@ -1,24 +1,389 @@
+> or refer to the [CVAT Course](https://www.youtube.com/playlist?list=PL0to7Ng4PuuYQT4eXlHb_oIlq_RPeuasN)
+> for installation videos.
+
+All analytics-related features will be launched when you
+start CVAT containers with the following command:
 
 ```shell
 docker compose up -d
 ```
 
-The previous solution based on ELK stack is currently deprecated and will no longer be supported.
+### Ports settings
+
+In case you cannot access analytics, check if the following ports are open:
+
+```json
+
+cvat_vector:
+    ports:
+      - '8282:80'
+
+  cvat_clickhouse:
+    ports:
+      - '8123:8123'
+```
+
+**Or** remove the following data from
+[launch.json](https://github.com/opencv/cvat/blob/develop/.vscode/launch.json):
+
+```json
+  "DJANGO_LOG_SERVER_HOST": "localhost",
+  "DJANGO_LOG_SERVER_PORT": "8282"
+```
+
+Analytics on GitHub:
+[Analytics Components](https://github.com/opencv/cvat/tree/develop/components/analytics)
+
+### Events Log structure
+
+[Relational database](https://github.com/opencv/cvat/blob/develop/components/analytics/clickhouse/init.sh)
+schema with the following fields:
+
+<!--lint disable maximum-line-length-->
+
+| Mandatory Field | Description                                                                             |
+| --------------- | --------------------------------------------------------------------------------------- |
+| scope           | scope of the event (e.g., zoomin:image, add:annotations, delete:image, update:assignee) |
+| obj_name        | object name or None (e.g., task, job, cloudstorage, model, organization)                |
+| obj_id          | object identifier as in DB or None                                                      |
+| obj_val         | value for the event as string or None (e.g., frame number, number of added annotations) |
+| source          | who generates the log event (e.g., server, ui)                                          |
+| timestamp       | local event time (in general for UI and server, the time is different)                  |
+| count           | how many times in the row it occurs                                                     |
+| duration        | how much time does it take (it can be 0 for events without duration)                    |
+| project         | project ID or None                                                                      |
+| task            | task ID or None                                                                         |
+| job             | job ID or None                                                                          |
+| user            | user ID or None                                                                         |
+| organization    | organization ID or None                                                                 |
+
+<!--lint enable maximum-line-length-->
+
+Extra fields can be added to the JSON blob.
```

Comment:
This is the description for the missing field `payload`. For example: payload is astringified JSON, which can contain any additional data.